server/app/routes/api/wines.js

```diff
@@ -0,0 +1,102 @@
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose')
+
+router.get('/', function(req, res) {
+	mongoose.model('Wine')
+	.find({})
+	.then(function(wines) {
+		res.json(wines)
+	})
+	.then(null, function(err) {
+		console.log(err)
```

Comment:
probably better to use next - as of now, errors will be caught and logged here, but won't make their way to the error handler in routes/index